docs/primer/releases_and_records.md

```diff
@@ -38,7 +38,7 @@ Each time a new release is published it is added to the index, the compiled rele
 
 ![A contracting (or planning) process is described by many releases, which are aggregated into a single record](../_static/png/change_history_process_record.png)
 
-Records and releases each contain several fields which can be used in different sections. The OCDS schema sets out the fields that ought to be included in each section (where applicable), aiming to reuse simple structures to represent information. For example, a release may contain information about items being procured. OCDS' schema sets out standard building blocks for items across releases, including the name of the item, a description, each item's value, and the currency used.
+Records and releases each contain several fields which can be used in different sections. The OCDS schema sets out the fields that ought to be included in each section (where applicable), aiming to reuse simple structures to represent information. For example, a release may contain information about items being procured. OCDS’ schema sets out a standard subschema for items across releases, including the name of the item, a description, each item’s value, and the currency used.
```

Comment:
I initially tried to rephrase to avoid mentioning subschema, but then I realised that I find this whole paragraph very hard to parse so I took a pass at rewording it based on what I think it is trying to communicate:

```suggestion
Some concepts are common to several stages of the contracting (or planning) process. The OCDS schema provides reusable subschemas to represent common concepts. For example, the [`Item` subschema](../schema/reference.md#item) represents the items to be procured at the tender stage and the items to be contracted at the contract stage. It contains fields applicable to either stage, such as an item's title, description and value.
```

Reply:
You're right that this was a tough paragraph to parse. The example of `items` however is incomplete, `items` can also be used for the items awarded at the award stage, so I've added this in https://github.com/open-contracting/standard/pull/1660/commits/198563675ce1585789d701576ab4873c92840899

Reply:
Sorry to nitpick, but I think my suggested sentence is clearer because it illustrates two sets of items (the items to be procured and the items to be contracted) that are represented using the `Items` subschema. Your sentence is more ambiguous because it could be read as there being one set of items which are procured, awarded and contracted. I didn't mention the award stage because the example is intended to be illustrative of the reuse of a subschema rather than a comprehensive list of the use of items in the schema. Likewise, I only mentioned title, description and value rather than listing all of the properties of items.

Reply:
no apology necessary but I'm going to nitpick too :) I read the way you're phrased it as saying that 'items' can only be used in 'tender' and 'contracts' whereas it can be used in 'awards' as well, this was all I was trying to make clear. Maybe it just needs a comma to be clearer in your version to change the emphasis... 
change made in [585e1a16](https://github.com/open-contracting/standard/commit/585e1a1655d548853ab7ee4c7b628e3c40e835be)